server/lib/template/helpers/finance.js

```diff
@@ -21,7 +21,7 @@ function currency(value, currencyId) {
   // @TODO - super-hardcoded values for the moment.  Can we do better?
   const fmt = (currencyId === 1) ? FC_FMT : USD_FMT;
 
-  return accountingjs.formatMoney(value, fmt);
+  return value ? accountingjs.formatMoney(value, fmt) : accountingjs.formatMoney(0, fmt);
```

Comment:
Hmm...  What this code says to me is "if the value is 0, null, false, or undefined, default to 0".  Is that what you mean?  It seems like we will miss lots of bugs because of this.